packages/react-native/src/Authenticator/Defaults/SetupTOTP/__tests__/SetupTOTP.spec.tsx

```diff
@@ -96,13 +97,14 @@ describe('SetupTOTP', () => {
   });
 
   it('handles secret code generation as expected', async () => {
-    const { queryByText, rerender } = render(
+    const { getByText, queryByText, rerender } = render(
       <SetupTOTP {...props} getTotpSecretCode={mockGetTotpSecretCode} />
     );
     await waitFor(async () => {
       expect(mockGetTotpSecretCode).toHaveBeenCalledTimes(1);
+      expect(getByText(getSetupTOTPInstructionsText())).toBeDefined();
       expect(queryByText(SECRET_KEY)).toBeDefined();
-      expect(queryByText(SECRET_KEY)?.props.selectable).toBe(true);
+      expect(getByText(SECRET_KEY).props.selectable).toBe(true);
```

Comment:
Just for my own understanding, why use both `queryByText` and `getByText`? Is it that `queryByText` can be used to check for elements that might be undefined?

Reply:
Correct, `getByText` throws an error if the element isn't foun, while `queryByText` returns undefined. Can use either to check for element but line 107 would have been problematic as it doesn't really test anything if the element is undefined.